dhcpv4/option_domain_name_server.go

```diff
@@ -0,0 +1,71 @@
+package dhcpv4
+
+import (
+	"fmt"
+	"net"
+)
+
+// This option implements the domain name server option
+// https://tools.ietf.org/html/rfc2132
+
+// OptDomainNameServer represents an option encapsulating the server identifier.
```

Comment:
server identifier -> domain name servers

Reply:
silly me. Good catch! I used another option file as template and didn't replace everything